pd-control.md

```diff
@@ -804,6 +804,8 @@ Usage:
 >> scheduler add grant-leader-scheduler 1         // Schedule all the leaders of the Regions on store 1 to store 1
 >> scheduler add evict-leader-scheduler 1         // Move all the Region leaders on store 1 out
 >> scheduler config evict-leader-scheduler        // Display the stores in which the scheduler is located since v4.0.0
+>> scheduler config evict-leader-scheduler add-store 2    // Add leader eviction scheduling for store 2
```

Comment:
Do we need to align the comment?

Reply:
https://github.com/pingcap/docs/pull/17968/commits/7bd5ada8b2dacec88c838515aabc4f0c67bca643